code/modules/mob/inventory.dm

```diff
@@ -149,7 +149,7 @@
 	if (src.client)
 		src.client.screen -= I
 	I.layer = initial(I.layer)
-	I.plane = initial(I.plane)
+	I.set_plane(initial(I.plane))
```

Comment:
set_plane seems to only be used in a few cases, while many examples of directly setting the plane remain.
Why so?

Reply:
There is not so many places where plane is assigned, excluding object definitions. I changed almost all of them.